frontend/src/entries.ts

```diff
@@ -15,11 +16,19 @@ import {
 export interface Posting {
   account: string;
   amount: string;
+  meta: EntryMetadata | null;
```

Comment:
I think dropping the `| null` here and ensuring that this is always set in the frontend code even if not set in the Beancount entries might simplify reuse of the existing components

Reply:
This should be fixed in https://github.com/beancount/fava/pull/1722/commits/b851eb3ee76617e2d00b63980489f354356777a2 :)